### PR TITLE
ArcballControls: Remove more clone() calls.

### PR DESCRIPTION
**Pull request by gero3**
Related issue: #22823

**Description**

This PR removes some unnecessary clones from the arcballControls.

### CODE REVIEW

File: examples/jsm/controls/ArcballControls.js [hunk 1]
```diff
@@ -67,7 +67,9 @@ const _endEvent = { type: 'end' };
 const _raycaster = new Raycaster();
 const _offset = new Vector3();
 
-
+const gizmoMatrixStateTemp = new Matrix4();
```
Comment (Mugen87) [inline examples/jsm/controls/ArcballControls.js:70]: Would you mind using this pattern?
```js
const _gizmoMatrixStateTemp = new Matrix4();
```
Module scope variable should use the `_` prefix.

File: examples/jsm/controls/ArcballControls.js [hunk 2]
```diff
@@ -2006,11 +2008,11 @@ class ArcballControls extends EventDispatcher {
 		_offset.copy( point ).sub( this._gizmos.position ).multiplyScalar( amount );
 		this._translationMatrix.makeTranslation( _offset.x, _offset.y, _offset.z );
 
-		const gizmoStateTemp = this._gizmoMatrixState.clone();
+		_gizmoMatrixStateTemp.copy( this._gizmoMatrixState.copy() );
```
Comment (Mugen87) [inline examples/jsm/controls/ArcballControls.js:2011]: Shouldn't it be just:
```js
_gizmoMatrixStateTemp.copy( this._gizmoMatrixState );
```

Comment (gero3) [reply examples/jsm/controls/ArcballControls.js:2011]: yes

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**